shotover-proxy/src/frame/cassandra.rs

```diff
@@ -322,6 +353,7 @@ pub enum CassandraResult {
         value: MessageValue,
         metadata: RowsMetadata,
     },
+    // SetKeyspace and Prepared are boxed because they take up a lot more stack space than Void.
```

Comment:
Rows will also take up more stack space, just Clippy doesn't know it yet.

Reply:
Do you mean there is a bug in clippy?
Or the stackspace usage isnt big enough to trigger the clippy lint?

Reply:
I was thinking when the value is filled with `MessageValue::Rows` but I just realised it will all be in a heap allocated vec.